index.bs

```diff
@@ -10366,13 +10368,11 @@ is the [=interface=]:
         |I|.
 </ol>
 
-If the internal \[[Call]] method
-of the [=interface object=]
-returns normally, then it must
-return an object that implements interface |I|.
+If evaluating the [=function object=] |F| returns normally,
+then it must return an object that implements interface |I|.
 This object also must be
 associated with the ECMAScript global environment associated
-with the interface object.
+with |I|.
```

Comment:
I guess maybe just fix https://github.com/heycam/webidl/issues/206 at the same time.